compiler/src/test/java/com/squareup/anvil/compiler/testing/AnvilAnnotationsTest.kt

```diff
@@ -0,0 +1,56 @@
+package com.squareup.anvil.compiler.testing
+
+import com.rickbusarow.kase.DefaultTestEnvironment
+import com.rickbusarow.kase.Kase1
+import com.rickbusarow.kase.KaseTestFactory
+import com.rickbusarow.kase.ParamTestEnvironmentFactory
+import com.rickbusarow.kase.files.HasWorkingDir
+import com.rickbusarow.kase.files.TestLocation
+import com.rickbusarow.kase.kases
+import com.squareup.anvil.compiler.isFullTestRun
+import kotlin.reflect.KClass
+
+abstract class AnvilAnnotationsTest(
+  firstAnnotation: KClass<out Annotation>,
```

Comment:
Rename to something like `requiredAnnotation`? To help signify what it's used for and why it's separate from `fullTestRunAnnotations`

Reply:
Done: https://github.com/square/anvil/pull/1033/commits/4df2fb9e48bc26d9b0a30ec5058b0b023408b9ce